src/app/config/icon_cfg.rs

```diff
@@ -21,43 +21,43 @@ impl Default for IconsConfig {
 
 fn get_default_dir_icons() -> HashMap<String, String> {
     let mut icon_map = HashMap::new();
-    icon_map.insert(".git".to_string(), "".to_string());
-    icon_map.insert("node_modules".to_string(), "".to_string());
-    icon_map.insert("default".to_string(), "".to_string());
+    icon_map.insert(".git".to_string(), "📁".to_string());
```

Comment:
This is some default icon/emoji ?

Reply:
yes, by default unicode emojis are rendered in Rust. So we can just copy-paste them in.

Reply:
Ok seems legit I want to know because I see this for the first time. Thanks and good job :+1: